src/OrchardCore/OrchardCore.UrlRewriting.Core/Models/UrlRedirectSourceMetadata.cs

```diff
@@ -4,19 +4,19 @@ public class UrlRedirectSourceMetadata
 {
     public string Pattern { get; set; }
 
-    public bool IgnoreCase { get; set; }
-
     public string SubstitutionUrl { get; set; }
 
+    public bool IgnoreCase { get; set; }
+
     public bool AppendQueryString { get; set; }
 
     public RedirectType RedirectType { get; set; }
 }
 
 public enum RedirectType
 {
-    Found302,
-    MovedPermanently301,
-    TemporaryRedirect307,
-    PermanentRedirect308,
+    Found = 302,
```

Comment:
How this change deals with default enum value if user don't provide one for example when creating from recipe? We would like 302 to be default.

Reply:
check out the last commit